modules/photo/test/test_decolor.cpp

```diff
@@ -43,8 +43,8 @@
 
 namespace opencv_test { namespace {
 
-TEST(Photo_Decolor, regression)
-{
+    TEST(Photo_Decolor, regression)
+    {
```

Comment:
No need to add indentation into namespaces.
Please revert changes in this file.

Reply:
Other tests are indented by 4 spaces but this code is indented by 8 spaces. I decrease indentation of this code. 
https://github.com/opencv/opencv/blob/master/modules/photo/test/test_denoise_tvl1.cpp#L83